jujugui/static/gui/src/app/components/login/login.js

```diff
@@ -82,6 +82,9 @@ class Login extends React.Component {
   }
 
   _generateUSSOLink() {
+    if (!this.props.bakeryEnabled) {
+      return '';
```

Comment:
We will want to return null or undefined here not an empty string.

Reply:
Done.